bridge/opentracing/bridge_test.go

```diff
@@ -574,3 +574,89 @@ func TestBridgeSpanContextPromotedMethods(t *testing.T) {
 		assert.True(t, spanContext.(spanContextProvider).HasTraceID())
 	})
 }
+
+func TestBridgeCarrierBaggagePropagation(t *testing.T) {
+	testCases := []struct {
+		name             string
+		format           ot.BuiltinFormat
+		carrier          interface{}
+		baggageItems     []bipBaggage
+		wantBaggageItems []bipBaggage
+	}{
+		{
+			name:    "HTTPHeadersCarrier",
+			format:  ot.HTTPHeaders,
+			carrier: ot.HTTPHeadersCarrier(http.Header{}),
```

Comment:
unless you envision different test cases for HTTP vs. Text, I would pull the type of carrier into another pre-defined loop, rather than being part of the test case.